compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -1674,6 +1678,10 @@ convertTyCon env t
                 pure $ if envLfVersion env `supports` featureTypeRep
                     then TTypeRep
                     else TUnit
+            "AnyException" -> pure (TBuiltin BTAnyException)
+            "GeneralError" -> pure (TBuiltin BTGeneralError)
```

Comment:
probably too late now and doesn’t matter all that much but did we consider calling this `UserError` instead? General sounds a bit like it is the supertype of all exceptions.

Reply:
The names/types/values of primitive exceptions are not final :) We decided to just go ahead with these for now and leave the final decisions (& bikeshedding) for later.

Reply:
sounds very sensible :slightly_smiling_face: